packages/suite/src/components/firmware/CheckSeed.tsx

```diff
@@ -6,10 +6,8 @@ import { Translation } from '@suite-components';
 import { OnboardingStepBox } from '@onboarding-components';
 import { P } from '@firmware-components';
 
-const CheckboxRow = styled.div`
-    display: flex;
-    flex-direction: row;
-    justify-content: center;
+const StyledChechbox = styled(Checkbox)`
```

Comment:
Chech :)

Reply:
Paying my respects to the culture!

Reply:
cbd767025bd9279bc2d96c7ad5381c64bd6c3767